### DISCUSSION TITLE
Galaxy theme does not fill a 1440p screen

### DISCUSSION THREAD

**Discussion by appuchias** [JulianPrieber/littlelink-custom#353] [Themes]
The "stars" of the theme don't fill the whole page, leaving the right side without any star, just the gradient background.

I've traced it to the skeleton-auto.css file from the galaxy theme, but I don't know how to modify it properly.

Screenshot:
![The right side of the page does not show stars](https://user-images.githubusercontent.com/46938448/222827072-c303296e-30b2-4925-8035-0b6a60ffc3c0.png)

**Comment by CamdenJaxx** [comment]
Hey there,

Try replacing the skeleton-auto.css file with this one; this should fill the screen for you.

[skeleton-auto.css.txt](https://github.com/LinkStackOrg/LinkStack/files/11500531/skeleton-auto.css.txt)

**Comment by appuchias** [comment]
In case anyone comes back to this discussion, this has been solved as of the last Galaxy theme update.

To be precise, in [this commit](https://github.com/LinkStackOrg/galaxy/commit/d06f6af27b9ab5cfd52f0a0f231c74a3ef4abc81)
